backend/LexBoxApi/Services/ProjectService.cs

```diff
@@ -38,6 +39,7 @@ public async Task<Guid> CreateProject(CreateProjectInput input)
                 IsConfidential = isConfidentialIsUntrustworthy ? null : input.IsConfidential,
                 Organizations = theOrg is not null ? [theOrg] : [],
                 Users = input.ProjectManagerId.HasValue ? [new() { UserId = input.ProjectManagerId.Value, Role = ProjectRole.Manager }] : [],
+                FlexProjectMetadata = input.Type == ProjectType.FLEx ? new() : null
```

Comment:
I did this because CrdtCommit references the FlexProjectMetadata table, and so if the row is missing then the commits can't be added even though it's a FLEx project, maybe I should change that, but it seemed like a decent way to ensure that they can only reference FLEx projects